src/domain/challenge/common/HubChildJourneyCard/JourneyCardParentSegment.tsx

```diff
@@ -20,11 +22,12 @@ const JourneyCardParentSegment = ({
     <CardSegmentCaption
       component={RouterLink}
       to={parentJourneyUri}
-      icon={<Icon fontSize="small" color="primary" />}
-      align="left"
+      icon={Icon ? <Icon fontSize="small" color="primary" /> : undefined}
+      secondaryIcon={locked ? <LockOutlined fontSize="small" color="primary" /> : undefined}
       onClick={stopPropagation}
+      disablePadding
     >
-      <Caption noWrap>{children}</Caption>
+      <>{children}</>
```

Comment:
Here I have to pass a node because `CardSegmentCaption` ends up printing `children` in a `<Caption>` which doesn't accept `undefined`